addons/web_editor/views/snippets.xml

```diff
@@ -444,7 +444,7 @@
         <div class="o_we_image_shape">
             <we-row string="Shape" class="o_we_full_row">
                 <we-select-pager data-name="shape_img_opt" class="o_we_select_grid o_we_fake_transparent_background">
-                    <we-select-page string="Basics">
+                    <we-select-page string="Basics" t-if="not no_animations">
```

Comment:
I don't like when we use `not` and then a boolean variable which has a negative name. It's very confusing. In this particular case it's even weirder because we test the value of an hypothetical variable which in fact does not exist, except in mass_mailing where it is defined. Is there no better way we can have this option disabled in mass_mailing ?

Reply:
That's the best way I could find. If I let it render then hide the animated shapes, we end up with empty pages or wrong numbering. I can't go and put yes_animations everywhere else because it's in different modules and what I want to do here is just disable the feature in mass_mailing. There are other places where this same thing is done, for instance no_transparency for the colorpicker.

Reply:
`yes_animations` 😎  Alright, I'm resolving this.